torch/distributed/tensor/_ops/_tensor_ops.py

```diff
@@ -759,9 +759,11 @@ def stack_strategy(op_schema: OpSchema) -> StrategyType:
     input_tuple_strategy = args_schema[0]
     if not isinstance(input_tuple_strategy, TupleStrategy):
         raise AssertionError(f"Expected TupleStrategy, got {input_tuple_strategy}")
-    first_input_strategy = input_tuple_strategy.children[0]
-    if not isinstance(first_input_strategy, OpStrategy):
-        raise AssertionError(f"Expected OpStrategy, got {first_input_strategy}")
+    input_strategies: list[OpStrategy] = []
```

Comment:
this part was just to make mypy happy below since children are listed as 'StrategyType' which can be Tuple Strategy or OpStrategy,  need to ensure they are all  OpStrategy...